src/Apps/Components/Layouts/LayoutLogoOnly.tsx

```diff
@@ -16,13 +16,13 @@ export const LayoutLogoOnly: FC<BaseLayoutProps> = ({ children }) => {
       <LayoutMain>
         <AppContainer>
           <HorizontalPadding>
-            <Spacer y={4} />
+            <Spacer y={2} />
```

Comment:
This isn't something that should be changed.

Reply:
It was ticketed here: https://artsyproduct.atlassian.net/browse/TX-1079

Reply:
This changes it for the entire app though.

Reply:
The whole point of using a layout is consistency. I feel like we're missing something here

Reply:
Yeah, I interpreted the ticket as updating the logo-only layout for everywhere on the app so the checkout is consistent with everything else. I'm going to clarify this.

Reply:
Otherwise we need to create another layout

Reply:
> Otherwise we need to create another layout

I would disagree with this :)

Why would we want an inconsistent experience in checkout? What's the underlying _intent_ behind this change? There's probably a better path forward.

Reply:
For instance: I actually think this _should_ be `<Spacer y={[2, 4]} />` since we want that correspondence with the outer margins.

Reply:
I definitely agree that the checkout should use the `LogoOnly` layout, but design wants the spacing changed. I talked with Viviana and she is okay with the 20px header spacing on the error pages / other pages using `LogoOnly`. So we can go ahead with this spacing change if it's okay with you.

Reply:
RIP to this perfect correspondence:

![rip](https://user-images.githubusercontent.com/112297/226007142-b08dd01b-c077-495f-afba-c45cf1f064a8.jpg)

Reply:
@rquartararo - We should push back on this. Perhaps Viviana isn't aware of the wider implications of this change?

Reply:
I mean: I'd push back just because I think it looks nicer and we tend to use `[2, 4]` on _every_ page as the top margin. But the implications aren't _that_ wide. The layout is only used on error pages and I guess now the checkout. I do think it's going to look awkward on the error page and that if the goal is to fit the checkout on a single screen, misguided.

Reply:
Its wide, because an error can throw from any page! so immediately that's obvious (and doesn't feel necessary).

Reply:
Okay, Viviana agreed to table this change for now! I reverted it back to 40px. Let me know if all is good now and I can merge this.

Reply:
🙏 👍